### ISSUE TITLE
[Proj 4] permission to remove a file

### ISSUE THREAD

**Issue by soleee99** [swsnu/osspr2022#131]
안녕하세요, proj4 관련해서 질문이 있어서 이슈 남깁니다!

혹시 A 라는 {file/dir}의 location 과 현재 location 에 대해
is_geo_close 가 0인 경우에 (i.e. 가깝지 않은 경우)
현재 location 에서 A를 rm 하는 것도 불가능하도록 구현을 하는게 맞을까요?

보통 Permission 이라 함은 generic 하게 r w x 를 의미한다고 생각해서, 혹시 rm 를 따로 신경써줘야 하는지 궁금합니다.
감사합니다 : )

**Comment by t-hyun** [comment]
No, you don't have to care about the permission of A if you want to delete A!
since a user needs no permissions on A nor needs to be the A's owner to delete it.
(But you need (w + x) permission on a directory where A is located.)

**Comment by cakelemon** [comment]
@t-hyun 
안녕하세요!
저희 팀의 경우 파일 제거에도, 파일 자체에 대한 location permission 이 필요하도록 구현이 되어 있는데요,
이 경우 위 답변과 맞게 directory 권한만 필요하도록 수정해야 하나요?
감사합니다.

**Comment by widiba03304** [comment]
@cakelemon 파일을 제거하는 것은 그 파일이 들어있는 디렉토리를 수정하는 것과 같게 취급합니다!
ext 기반 파일 시스템에서는 따로 파일에 대한 삭제 권한을 두고 있지 않기 때문에 그렇지 않도록 하는 것이 맞을듯합니다!
